src/main/java/net/imagej/ops/image/ascii/DefaultASCII.java

```diff
@@ -115,6 +117,8 @@ public String calculate(final IterableInterval<T> input) {
 			// normalized = (value - min) / (max - min)
 			tmp.set(cursor.get());
 			tmp.sub(min);
+			// NB: if a value is below min we set tmp to zero.
+			if (tmp.compareTo(zero) < 0) tmp.setZero();
 			final double normalized = tmp.getRealDouble() / span.getRealDouble();
```

Comment:
As long as we are bending over backwards to use the ImgLib2 math APIs... why don't we use `tmp.div(span)`, and only afterward call `getRealDouble` on the result?

Reply:
Actually, this will not work if the type does not support decimal values, since we are normalizing the data. I will fix this Wednesday morning.